coalib/parsing/Globbing.py

```diff
@@ -216,9 +216,6 @@ def fnmatch(name, globs):
     """
     globs = (globs,) if isinstance(globs, str) else tuple(globs)
 
-    if len(globs) == 0:
-        return True
```

Comment:
Doesn't it make more sense to return False? If there aren't any globs, it's not a match.